include/deal.II/base/utilities.h

```diff
@@ -445,7 +447,9 @@ namespace Utilities
    * @author Timo Heister, Wolfgang Bangerth, 2017.
    */
   template <typename T>
-  size_t pack (const T &object, std::vector<char> &dest_buffer);
+  size_t pack (const T &object,
+               std::vector<char> &dest_buffer,
+               const bool force_uncompressed = false);
```

Comment:
Double negatives ("UNcompressed" and "false") are often difficult to understand. How about call this parameter "allow_compression", with a default of "true"? That would be easier to grasp, I think.